src/components/SettingsMenu.js

```diff
@@ -57,7 +57,9 @@ const EditorPanel = () => {
 
       </Tabs>
 
-      {tabName === 'Export' && <ExportEditor />}
+      <Suspense fallback={<div>loading...</div>}>
+        {tabName === 'Export' && <ExportEditor />}
```

Comment:
you may as well lazy load all of the tabs

Reply:
I tried doing for tabs, but no good since `TABS` object is accessed to before rendering so can't really lazy load it

Reply:
You need to simply just wrap each tab in a suspense, not the entire `TABS.map`.